templates/batch-llm/README.ipynb

```diff
@@ -93,31 +147,44 @@
     "The key to a happy life is ...\n",
     "\"\"\",\n",
     "]\n",
-    "ds = ray.data.from_items(prompts)"
+    "ds = ray.data.from_items(prompts)\n",
+    "\n",
+    "# View one row of the Dataset.\n",
+    "ds.take(1)"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "NOTE: This notebook uses local text data defined above, but it can easily be expanded to read from input data stored in cloud storage. To read text files from cloud storage (e.g., AWS S3), you can modify the dataset to be:\n",
+    "### Scaling to a larger dataset\n",
+    "In the cell above, we created a Ray Dataset with 5 example prompts. Next, let's explore how to scale to a larger dataset based on files stored in cloud storage.\n",
     "\n",
-    "```python\n",
-    "ds = ray.data.read_text(\"s3://anonymous@air-example-data/prompts.txt\")\n",
-    "```"
+    "Run the following cell to create a Dataset with 100 example prompts, constructed by repeating the 5 examples above 20 times."
```

Comment:
why change to repeating the synthetic data, instead of using cloud data? 
Cloud data would be closer to the real use case. 
I'm not strongly against it tho. if we decide to use synthetic data, maybe also add a note to let readers know about the `read_text/read_jsonl` APIs